snowflake/datadog_checks/snowflake/check.py

```diff
@@ -103,14 +102,19 @@ def connect(self):
                 client_prefetch_threads=self.config.client_prefetch_threads,
                 login_timeout=self.config.login_timeout,
                 ocsp_response_cache_filename=self.config.ocsp_response_cache_filename,
+                authenticator=self.config.authenticator,
+                token=self.config.token,
+                client_session_keep_alive=True,
```

Comment:
Since we don't have integration/e2e tests. I think it would be nice to have a unit tests asserting that init/instance configs are correctly passed to `sf.connect()` (also true for existing configs like user/password/account, etc).

Reply:
I've added additional tests for these options in `tests/test_unit.py`